morphic.js

```diff
@@ -7818,10 +7820,47 @@ MenuMorph.prototype.select = function (aMenuItem) {
     this.selection = aMenuItem;
 };
 
+MenuMorph.prototype.mouseLeave = function () {
+    if (this.parent instanceof MenuMorph) {
+        // this is a submenu so we vanish when lose focus
+        var mouses = this.world.hand.allMorphsAtPointer();
+        if (contains(mouses, this.parent)) {
+            var items = mouses.filter(function (morph)
```

Comment:
@liam4 Anyone who uses names such as x and y in programming (other than in a procedure that plots graphs of functions) deserves whatever happens, regardless of where declared.  Come up with an example in which someone using self-documenting variable names will be burned that way.